css/src/components/hero.scss

```diff
@@ -140,16 +184,26 @@ $hero-image-width-widescreen: calc((100% - 2 * $layout-widescreen-gap) * 0.55) !
 
 	// Set direction to prevent image flip for rtl
 
+	[dir='rtl'] &.hero-image,
```

Comment:
What is this selector adding that the `&.hero-image:dir(rtl)` below isn't adding?

I ask because using an attribute selector for direction has a weird folly in that this matches:

```html
<html dir="rtl">
    <body dir="ltr">
        <section class="hero hero-image">...</section>
    </body>
</html>
```

This matches, since `.hero.hero-image` has any rtl ancestor, even though a closer ancestor is defined as ltr.

`:dir(rtl)` should accurately match the element's actual ltr/rtl state.

Reply:
I think we're good to remove this as we're not flipping anything.  @wibjorn  are you good with me removing that?

Reply:
Context here was https://github.com/microsoft/atlas-design/pull/616#discussion_r1567948781 - resolving as a result